guides/common/modules/con_comparing-multiple-content-view-environments-and-composite-content-views.adoc

```diff
@@ -0,0 +1,20 @@
+[id="comparing_multiple_content_view_environments_and_composite_content_views_{context}"]
+= Comparing multiple content view environments and composite content views
+
+Composite content views provide an alternative method for granting hosts access to content from multiple content views.
+You can use composite content views, multiple content view environments, or a combination of both.
+The key differences between these methods include the following:
+
+* *Access Timing*:
```

Comment:
```suggestion
Access Timing::
```

Suggestion (non-blocking): Use asciidoc glosssary items or whatever it's called. Refs https://docs.asciidoctor.org/asciidoc/latest/sections/glossary/#glossary-description-list-style-syntax and https://github.com/theforeman/foreman-documentation/blob/master/guides/common/modules/con_content-flow-in-project.adoc?plain=1#L9

Reply:
same for line 12 and line 16 in this file.